src/server/src/resources/emails.js

```diff
@@ -39,3 +39,17 @@ export const generateForgotPasswordEmail = (email, name, token) => {
             The Lecture Website App Generator team</p>`,
   };
 };
+
+export const generateNewNotActivatedEmail = (email, name, token) => {
+  return {
+    from: "Lecture Website App Generator <eyalgolan96@gmail.com>",
+    to: email,
+    subject: `Hi ${name}, please verify your Lecture Website App Generator account`,
```

Comment:
Get this text from Ayelet

Reply:
I've contacted her about this in the past. I'll check with her again.